website/secrethub.erb

```diff
@@ -25,6 +25,7 @@
           <ul class="nav nav-visible">
             <li<%= sidebar_current("docs-secrethub-resource-secret") %>>
               <a href="/docs/providers/secrethub/r/secret.html">secrethub_secret</a>
+              <a href="/docs/providers/secrethub/r/service.html">secrethub_service</a>
```

Comment:
This is not correct. `sidebar_current("docs-secrethub-resource-secret")` != `secrethub_service`.